### DISCUSSION TITLE
Cannot scan to the WiFI list

### DISCUSSION THREAD

**Discussion by esorylul** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#19] [Q&A] [answered]
Thank you very much for your EFI, the installation is smooth, but after the installation, I have a little problem. WIFI cannot be used. I tried to make corrections through online tutorials, but there is no response after many changes, so I will ask you.

**Comment by adityabakare** [comment, marked as answer]
What's your wifi card? The easiest way to check it is via Hackintool under PCIe. My current EFI supports only Intel cards, and I'm having no issues so far.

Moreover, there is a switch for turning wireless on/off, which can be triggered with Fn + PrtScr. Check that too.

**Reply by esorylul** [reply]
sorry! I'm stupid.  My wifi card is RTL8723BE. 🥲  forgive me.
